projects/1.16.1/assets/astral-sorcery/astralsorcery/lang/zh_cn.json

```
@@ -162,161 +162,161 @@
-  "astralsorcery.journal.ROCK_CRYSTALS.1.text": "由于产生它们的特殊环境，每种新开采的水晶石都是不同的。在尺寸、纯度与可塑性上的差异，都会导致对特定用途的适用性存在相当大的差异。在把它们从岩石层中挖掘出来前，没有如何办法可以确定这些水晶石的具体属性。虽然学院早已认知到对水晶石的修正是利用水晶石的必经之路，但这可不像给一个普通宝石整容那么简单。",
-  "astralsorcery.journal.ROCK_CRYSTALS.2.text": "将水晶石用磨石粗暴的打磨只会造成破坏。因此需要用更优雅的方式对水晶进行§8修正§r。",
+  "astralsorcery.journal.ROCK_CRYSTALS.1.text": "由于它们的产生环境较为特殊，所以每枚新开采的水晶石都各有不同。每枚水晶石在尺寸、纯度与可塑性上的差异，都会导致他们在特定目的的适用性上产生相当大的差异。在把它们从岩石层中挖掘出来前，没有办法确定这些水晶石的具体属性。虽然学院早已认知到对水晶石的处理是利用水晶石的必经之路，但这可不像给一个普通宝石整容那么简单。",
+  "astralsorcery.journal.ROCK_CRYSTALS.2.text": "将水晶石用磨石粗暴的打磨只会造成破坏。因此需要用更精致的方式对水晶进行§8处理§r。",
 
-  "astralsorcery.journal.node.CRYSTAL_GROWTH.name": "水晶的修正",
-  "astralsorcery.journal.CRYSTAL_GROWTH.1.text": "基于内部矩阵基质受星能影响而进行强化重构的原理，将水晶石浸没在星能液中，直到星能液被吸收掉，可以修复或者让水晶石变得更坚固耐用。当发生这种情况时，水晶石的尺寸将增长。但任何水晶石的属性修正与属性数量的修正都有其极限，若不使用专用工具是无法删除水晶石上的属性的。",
-  "astralsorcery.journal.CRYSTAL_GROWTH.2.text": "将两个晶体浸入星能液中会导致属性较少的水晶石被合并到具有更多属性的水晶石中，“较大”的水晶石可能获得被吸收的水晶石的一个或多个属性。水晶石拥有的属性越多，发生这种情况的可能性就越小。",
+  "astralsorcery.journal.node.CRYSTAL_GROWTH.name": "水晶石处理",
+  "astralsorcery.journal.CRYSTAL_GROWTH.1.text": "水晶石会受到浓缩星光中的魔法能量的影响，进而发生内部矩阵的强化重排，所以将水晶石浸没在星光液并等待其将星光液完全吸收掉，可以将其修复或者变得更坚固耐用。当吸收完毕后，水晶石的尺寸将增长。任何水晶石的属性与属性数量的增幅都有其极限，同时，若不使用专用工具，是无法删除水晶石上的属性的。",
+  "astralsorcery.journal.CRYSTAL_GROWTH.2.text": "将两个晶体浸入星光液中会导致属性较少的水晶石被合并到具有更多属性的水晶石中，“较大”的水晶石可能获得被吸收的水晶石的一个或多个属性。水晶石拥有的属性越多，发生这种情况的可能性就越小。",
 
   "astralsorcery.journal.node.CUTTING_TOOL.name": "星辉锭切削工具",
-  "astralsorcery.journal.CUTTING_TOOL.1.text": "学院文献表明，虽然铁或钢凿子是一种很好的工具，但它仅是物理上的，而引导星能则完全相反，仅是魔法上的。将星辉锭的坚固特性与能量辐射场相结合，可以同时影响两种形态。金可以防止凿头的星能场变形，强化工具。<NL>对工具进行附魔将获得更好的保护，以防产生负面影响。",
-  "astralsorcery.journal.CUTTING_TOOL.3.text": "用工具敲击水晶石，可以分割水晶石以及其属性。新生成的水晶石将各自继承其父代属性的一部分，但这个过程既不均匀也不完美。有时还会导致部分属性消失。<NL>不断的将水晶石和雕凿出的碎片浸没在星能液中，再花费些时间，可以塑造出一个特定用途的水晶石，当然，这需要一些§5运气§r才行。",
+  "astralsorcery.journal.CUTTING_TOOL.1.text": "学院的文献表明，虽然铁或钢凿子用起来还算顺手，但这种“顺手”只能是停留在物理层面上，星光的引导则依赖于工具魔法层面的能力。星辉锭的坚硬和其自带的辐射魔法场域，可以同时满足切削工具对物理和魔法层面的双重需求。金则可以防止工具凿头的魔法场域发生变形，从而进一步增强了工具性能。<NL>对工具进行附魔将获得更好的保护，以防在使用中产生负面影响。",
+  "astralsorcery.journal.CUTTING_TOOL.3.text": "用工具敲击水晶石，可以分割水晶石以及其属性。新生成的水晶石将各自继承其父代属性的一部分，但这个过程既不均匀也不完美。有时还会导致部分属性消失。<NL>将水晶石和雕凿出的碎片反复浸没在星光液中，再花费些时间，便可以塑造出一个特定用途的水晶石，当然，这需要一些§5运气§r才行。",
 
   "astralsorcery.journal.node.TOOLS.name": "水晶石工具",
-  "astralsorcery.journal.TOOLS.1.text": "尽管特别容易受剪切应力影响，但精心特制的水晶石在导引背景星能时的性能要远远优于金属工具。使用多个水晶石专门制作的普通工具导引背景星能时，可以充分地利用其组成的水晶石所具有的属性并完美发挥出来。用这种方式制作的工具将拥有独特的作用，即使是同一为制作者所制造的工具也有可能因为所使用的水晶石而拥有完全不同的强度与能力。",
-  "astralsorcery.journal.TOOLS.3.text": "一些博学多识的提升者甚至可以找到在使用过程中为这些工具提供外部星能的方法，来进一步增强它们的功能，或以其他的方式利用它们的属性。",
+  "astralsorcery.journal.TOOLS.1.text": "水晶石不耐剪切，但若小心操作，如此制成的水晶石工具在导引背景星能时的性能要远远优于金属工具。在使用数个水晶石专门制作的普通工具导引背景星能时，可以充分地利用其组成的水晶石所具有的属性并完美发挥出来。用这种方式制作的工具将拥有独特的作用，即使是同一为制作者所制造的工具也有可能因为所使用的水晶石而拥有完全不同的强度与能力。",
+  "astralsorcery.journal.TOOLS.3.text": "若将受损的水晶石工具浸没到星光液中，那么当工具将星光液完全吸收之后，它们的耐久便会随之恢复。<NL>那些学识更丰富的学员甚至可以找到在使用过程中为这些工具提供外部星能的方法，来进一步增强它们的功能，或以其他的方式利用它们的属性。",

```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
  "astralsorcery.journal.TOOLS.3.text": "将受损的水晶石工具浸没到星光液中后，工具会完全吸收星光液，它们的耐久度也会随之恢复。<NL>那些学识更丰富的学员甚至可以找到在使用过程中为这些工具提供外部星能的方法，以进一步增强它们的功能，或用其他方式利用它们的属性。",

```